### PR TITLE
HDDS-3649. Add documentation for Cat Key Command

### PR DESCRIPTION
**Pull request by maobaolong**
## What changes were proposed in this pull request?

Add documentation for Cat Key Command

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-3649

## How was this patch tested?

No need, just docs changes.

### CODE REVIEW

**Comment by maobaolong** [comment]
@dineshchitlangia I add a document for cat key command, please take a look.

File: hadoop-hdds/docs/content/shell/KeyCommands.md [hunk 1]
```diff
@@ -137,3 +138,19 @@ The `key rename` command changes the name of an existing key in the specified bu
 ozone sh key rename /hive/jan sales.orc new_name.orc
 {{< /highlight >}}
 The above command will rename _sales.orc_ to _new\_name.orc_ in the bucket _/hive/jan_.
+
+### Cat
+
+The `key cat` command copies a specific Ozone key to standard output.
```
Comment (dineshchitlangia) [inline hadoop-hdds/docs/content/shell/KeyCommands.md:144]:
```suggestion
The `key cat` command displays the contents a specific Ozone key to standard output.
```

File: hadoop-hdds/docs/content/shell/KeyCommands.md [hunk 2]
```diff
@@ -137,3 +138,19 @@ The `key rename` command changes the name of an existing key in the specified bu
...
+
+| Arguments                      |  Comment                                |
+|--------------------------------|-----------------------------------------|
+|  Uri                           | The name of the key in **/volume/bucket/key** format.
+
+
+{{< highlight bash >}}
+ozone sh key cat /hive/jan/hello.txt
+{{< /highlight >}}
+Copies the file hello.txt from the _/hive/jan_ bucket to standard output.
```
Comment (dineshchitlangia) [inline hadoop-hdds/docs/content/shell/KeyCommands.md:156]:
```suggestion
Displays the contents of the key hello.txt from the _/hive/jan_ bucket to standard output.
```

**Comment by maobaolong** [comment]
@dineshchitlangia Thank you for your suggestion, PTAL.

File: hadoop-hdds/docs/content/shell/KeyCommands.md [hunk 3]
```diff
@@ -137,3 +138,19 @@ The `key rename` command changes the name of an existing key in the specified bu
 ozone sh key rename /hive/jan sales.orc new_name.orc
 {{< /highlight >}}
 The above command will rename _sales.orc_ to _new\_name.orc_ in the bucket _/hive/jan_.
+
+### Cat
+
+The `key cat` command displays the contents a specific Ozone key to standard output.
```
Comment (dineshchitlangia) [inline hadoop-hdds/docs/content/shell/KeyCommands.md:144]:
```suggestion
The `key cat` command displays the contents of a specific Ozone key to standard output.
```

**Comment by maobaolong** [comment]
@dineshchitlangia Thank you for your kindly review.

**Pull request merged**